exist-core/src/test/java/org/exist/util/serializer/HTML5WriterTest.java

```diff
@@ -42,7 +42,7 @@ public void setUp() throws Exception {
 
     @Test
     public void testAttributeWithBooleanValue() throws Exception {
-        final String expected = "<!DOCTYPE html>\n<input checked>";
+        final String expected = "<!DOCTYPE html><input checked>";
```

Comment:
would it be useful to have an other test with the \n still there, to show the (non?) effect of the changes?